EcoSoapBank/Features/Impact/ImpactViewController.swift

```diff
@@ -69,7 +69,10 @@ class ImpactViewController: UIViewController {
     // MARK: - Private Methods
     
     private func setUpCollectionView() {
-        collectionView.register(ImpactCell.self, forCellWithReuseIdentifier: ImpactCell.reuseIdentifier)
+        collectionView.register(
+            ImpactCell.self,
+            forCellWithReuseIdentifier: NSStringFromClass(ImpactCell.self)
```

Comment:
Probably doesn't make a difference, but if you wanted to avoid NSString I think you can also use `String(describing: ImpactCell.self)`.

Reply:
Ah, nice. I just saw the NSStringFromClass in Apple's example code and thought it was a good idea. Might make more sense to use String(describing:) though in the future.